apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js

```diff
@@ -1,10 +1,30 @@
+/* global appOptions */
```

Comment:
(minor) It seems like kind of a bummer to have to couple this code to appOptions.  Is there a better place to retrieve this (e.g. in a component that already depends on a lot of external state) and just pass it in here?

Reply:
I could pull it from redux pageConstants.locale and pass it in, I'll do that